dashboard/app/models/levels/level.rb

```diff
@@ -495,6 +495,16 @@ def summarize
     }
   end
 
+  def summary_for_feedback
+    {
+      lesson_name: script_levels[0].stage.name,
+      level_num: script_levels[0].position,
+      link_to_level: script_levels[0].path,
+      unit_name: script_levels[0].stage.script.localized_title,
+      link_to_unit: script_levels[0].stage.script.link
+    }
```

Comment:
@islemaster this is what I was referencing in sprint kickoff. This gets me the information that I want but I can't always trust that the first script_level is the accurate one. I think I need something like `script_levels.find_by(script: @script).stage.name`

Reply:
I see what you're saying.  This `summary` method definitely feels like it's pulling hard to live on `ScriptLevel` instead of `Level`, given that all the data it depends on lives there.

In some ways, this feels like an issue with `TeacherFeedback`.  Why is feedback tied to a `Level` and not a `ScriptLevel`? @epeach can you add some context to [the original design of this feature?](https://github.com/code-dot-org/code-dot-org/pull/22789/files#diff-95acc9aa84b45bf79b68a7bb970ebae0R23)

It's totally possible this is fallout from the new requirement that we can show feedback in a centralized place and link back to a script level, so we'll want to update our model accordingly. I'd definitely rather do that than try to infer the script level from context.